ReactNativeTemplate/android/app/src/main/java/com/salesforce/reactnativetemplate/MainApplication.java

```diff
@@ -67,14 +67,31 @@ public ReactNativeHost getReactNativeHost() {
 	@Override
 	public void onCreate() {
 		super.onCreate();
-		SalesforceReactSDKManager.initReactNative(getApplicationContext(), new ReactNativeKeyImpl(), MainActivity.class);
+		SalesforceReactSDKManager.initReactNative(getApplicationContext(), new ReactNativeKeyImpl(),
+                MainActivity.class);
+
+		/*
+         * Uncomment the following lines to enable IDP login flow. This will allow the user to
+         * either authenticate using the current app or use a designated IDP app for login.
+         * Replace 'idpAppURIScheme' with the URI scheme of the IDP app meant to be used.
+         */
+		// SalesforceReactSDKManager.getInstance().setIDPLoginFlowEnabled(true);
+		// SalesforceReactSDKManager.getInstance().setIDPAppURIScheme(idpAppURIScheme);
 
         /*
          * Uncomment the following line to enable browser based login. This will use a
          * Chrome custom tab to login instead of the default WebView. You will also need
          * to uncomment a few lines of code in SalesforceSDK library project's AndroidManifest.xml.
          */
         // SalesforceReactSDKManager.getInstance().setBrowserLoginEnabled(true);
+
+        /*
+		 * Un-comment the line below to enable push notifications in this app.
```

Comment:
Not sure why the `ReactNative` template was missing the GCM registration piece for push notifications, but I added it in.